offer-manager-app/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -9,7 +9,7 @@
 	<portlet>
 		<description>${project.description}</description>
 		<portlet-name>${project.artifactId}</portlet-name>
-		<display-name>${project.name}</display-name>
+		<display-name>${project.name} [${project.artifactId}]</display-name>
```

Comment:
What happens here?

Reply:
Ok.. got it :D so you just updated the portlet name so that the artifact id is reflected in the display name

Reply:
yes, i tried, but nothing happens :(

Reply:
maybe it is because of the brackets? 🤔

Reply:
should we remove the change if it is not working?